src/fragmentarium/ui/info/Details.tsx

```diff
@@ -89,6 +89,33 @@ function CdliNumber({ fragment: { cdliNumber } }: Props): JSX.Element {
   )
 }
 
+function EditedInOraccProject({
+  fragment: { editedInOraccProject, cdliNumber },
+}: Props): JSX.Element {
+  let projectLink = `http://oracc.org/${encodeURIComponent(
+    editedInOraccProject.toLowerCase()
+  )}/${encodeURIComponent(cdliNumber)}`
+  if (editedInOraccProject === 'ccp') {
+    projectLink = `https://ccp.yale.edu/${encodeURIComponent(cdliNumber)}`
+  }
```

Comment:
```javascript
const encodedCdliNumber = encodeURIComponent(cdliNumber)
const projectLink = editedInOraccProject == 'ccp' ? `http://oracc.org/${encodeURIComponent(editedInOraccProject.toLowerCase())}/${encodedCdliNumber}`
:
`https://ccp.yale.edu/${enocdedCdliNumber}`
```
It's better not to use let if possible because it's is mutable (const is safer).

Reply:
Thanks, incorporated